ui/pages/settings/settings.component.js

```diff
@@ -353,7 +358,9 @@ class SettingsPage extends PureComponent {
           render={() => <AddNetwork />}
         />
         <Route exact path={SECURITY_ROUTE} component={SecurityTab} />
-        <Route exact path={EXPERIMENTAL_ROUTE} component={ExperimentalTab} />
+        {shouldRenderExperimentalTab ? (
+          <Route exact path={EXPERIMENTAL_ROUTE} component={ExperimentalTab} />
+        ) : null}
```

Comment:
```suggestion
        {shouldRenderExperimentalTab ?? (
          <Route exact path={EXPERIMENTAL_ROUTE} component={ExperimentalTab} />
        )}
```

Reply:
I believe some time ago we switched to the null format return, as I think it caused some rendering issues. Correct me if I am wrong @darkwing

Reply:
I tried promoting that practice but it isn't strictly adhered to.  I wouldn't block on it until we have a discussion as a team.